keras/ops/nn.py

```diff
@@ -1823,7 +1823,7 @@ def ctc_loss(target, output, target_length, output_length, mask_index=0):
     Args:
         target: A tensor of shape `(batch_size, target_max_length)` containing
             the true labels in integer format.
-        output: A tensor of shape `(batch_size, output_max_length, num_classes)`
+        output: A tensor of shape `(output_max_length, batch_size, num_classes)`
```

Comment:
That seems very counterintuitive given that all output tensors in Keras start with the batch dimension. Surely we should transpose?

Reply:
Definitely, this is fixed now. Thanks!